iGCS/Models/Craft/GCSCraftArduCopter.m

```diff
@@ -42,4 +67,8 @@ - (BOOL) isInGuidedMode {
     return self.heartbeat.custom_mode == APMCopterGuided;
 }
 
+- (NSString *) currentModeName {
+    return self.customModeNames[@(self.heartbeat.custom_mode)];
```

Comment:
Would prefer (for now) to remove customModeNames names, and instead see a call to mavCustomModeToString instead. Wait wait, hear me out... :-)

After we get the tests back and running, then refactor these tests to use the craft models , *then* plop the modes into each craft (and also limit the enums to the specific models - too easy to mistype them in the wrong model at present).  This was the refactoring that I held off in the ICS-236 ticket, for the same reason.